src/command_modules/azure-cli-network/azure/cli/command_modules/network/_format.py

```diff
@@ -79,6 +79,29 @@ def transform_local_gateway_table_output(result):
     return final_result
 
 
+def transform_vpn_connection_output(result):
+
+    if not result:
+        return result
+
+    from azure.mgmt.network.models import VirtualNetworkGatewayConnectionPaged
+    is_list = isinstance(result, VirtualNetworkGatewayConnectionPaged)
+    result = list(result) if is_list else [result]
```

Comment:
Is there a way to do this without importing the SDK? That would be preferred.
e.g. see `transform_vm_list` and `transform_vm` in VM module.